clients/src/main/java/org/apache/kafka/clients/admin/internals/DescribeStreamsGroupsHandler.java

```diff
@@ -259,22 +258,9 @@ private void handleError(
                 break;
 
             case GROUP_ID_NOT_FOUND:
-                // In order to maintain compatibility with describeConsumerGroups, an unknown group ID is
```

Comment:
Ah, I didn't realize that we did this for compatibility with `describeConsumerGroups` when we talked about it. I still think we should rather go for the proper way and expose the `GROUP_ID_NOT_FOUND`.

Reply:
The `DescribeConsumerGroupsHandler` exposes the `GROUP_ID_NOT_FOUND` as well. Maybe they changed the logic as well.